reporter/src/main/resources/templates/notice/default.ftl

```diff
@@ -25,11 +25,6 @@ The notice file generated by this template consists of the following sections:
 Excluded projects and packages are ignored.
```

Comment:
Commit message:
`when all license text had no content`. Is this correct english?

I'd change it to `when none of the license texts had any content`.

Reply:
> Is this correct english?

I believe it is, it's just inverse logic compared to yours 😁 Changed anyway.